sliceio/writer.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2020 GRAIL, Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package sliceio
+
+import (
+	"github.com/grailbio/bigslice/frame"
+)
+
+// Writer can write a frame to an underlying data stream.
+type Writer interface {
+	// Write writes f to an underlying data stream. It returns a non-nil error
+	// if there is a problem writing, and f may have been partially written.
+	Write(f frame.Frame) error
```

Comment:
Optional: `Write` not taking `context.Context` seems a little confusing since it doesn't mirror `Read`. It looks like this is an artifact of always using background context to conform to the `gob` API, which might be worth documenting.

Reply:
Good point; added.